### PR TITLE
Added comments for elements.hmtl and styles.css

### PR DESCRIPTION
**Pull request by DominikRuf**
I added comments to the lastest version of elements and styles

### CODE REVIEW

File: site/assets/styles/styles.css [hunk 1]
```diff
@@ -46,41 +70,80 @@ a {
...
+    Klasse für aktuell ausgewählte Seite
+    Unterer Rand: Orange
+*/
 .header .header-navigation li.active {
     border-bottom: 2px solid #FCC509;
 }
 
+/* 
+    Auflistungsinhalte hover:
+    Unterer Rand: Orange mit 85% Transparenz
```
Comment (TomasKostadinov) [inline site/assets/styles/styles.css]: Not correct, this is hex and not %

File: site/assets/styles/styles.css [hunk 2]
```diff
@@ -27,10 +41,20 @@ a {
     float: left;
 }
 
+/*
+    distance:
+    Klasse für oberen Abstand 100px (genutzt in Hero)
```
Comment (TomasKostadinov) [inline site/assets/styles/styles.css]: This is used in containers, heros have automatically a distance to top

**Review by TomasKostadinov — CHANGES_REQUESTED** [review]
Looks good otherwise. Thanks!

File: site/assets/styles/styles.css (hunk 2 above)
Comment (DominikRuf) [reply site/assets/styles/styles.css]: Resolved

**Review by TomasKostadinov — APPROVED** [review]

**Pull request merged**